README.md

```diff
@@ -51,14 +59,35 @@ queue.async {
     This task begins execution once the above one-second sleep completes.
     */
 }
-Task {
-    await queue.await {
-        /*
-        `async` context that can return a value or throw an error.
-        Executes after all other enqueued work is completed.
-        Work enqueued after this task will wait for this task to complete.
-        */
+await queue.await {
+    /*
+    `async` context that can return a value or throw an error.
+    Executes after all other enqueued work is completed.
+    Work enqueued after this task will wait for this task to complete.
+    */
+}
+```
+
+With a `FIFOQueue` you can easily execute asynchronous tasks from a nonisolated context in FIFO order:
+```swift
+func testFIFOQueueOrdering() async {
+    actor Counter {
+        func increment() -> Int {
+            count += 1
+            return count
+        }
+        var count = 0
+    }
+
+    let counter = Counter()
+    let queue = FIFOQueue()
+    for iteration in 1...100 {
+        queue.async {
+            let incrementedCount = await counter.increment()
+            XCTAssertEqual(incrementedCount, iteration) // always succeeds
+        }
     }
+    await queue.await { }
```

Comment:
Maybe clarify with a comment that we're ensuring all tasks complete before we end the test?

Reply:
I'll fix this in #9 which builds on top of this change.

Reply:
c4d455f